src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js

```diff
@@ -95,15 +97,15 @@
 
             if (filterParameters[vm.dimension.name].indexOf(filter) !== -1) {
                 filterParameters[vm.dimension.name].splice(filterParameters[vm.dimension.name].indexOf(filter.name), 1);
-                removeTagFromSelectedList({text:filter});
+                removeTagFromSelectedList({ text: filter });
             }
             else {
                 filterParameters[vm.dimension.name].push(filter);
-                if(!vm.dimension.selected){
+                if (!vm.dimension.selected) {
```

Comment:
there is no change in code..if u want to format the code...pls create separate PR